source/_integrations/notify.markdown

```diff
@@ -12,7 +12,11 @@ ha_domain: notify
 
 The `notify` integration makes it possible to send notifications to a wide variety of platforms. To use it you have to setup at least one notification target (notifier), check the [integrations list](/integrations/#notifications) for one that fits your use case.
 
-If you want to send notifications to the Home Assistant web interface, you may use the [Persistent Notification integration](/integrations/persistent_notification/).
+If you want to send notifications to the Home Assistant web interface, you may use the [Persistent Notification integration](/integrations/persistent_notification/). 
+
+Sending notification via [Home Assistant Companion Apps](https://companion.home-assistant.io/docs/notifications/notifications-basic) can extended functionality with for example [actionable notifications](https://companion.home-assistant.io/docs/notifications/actionable-notifications) or [critical notifications](https://companion.home-assistant.io/docs/notifications/critical-notifications).
```

Comment:
This belongs on the mobile app integration page?

This page merely described the entity component

Reply:
Agree. But that page is also really hard to find if you are looking for notifications. It does not filter on notifications in the integration list it's under other. 

Not sure how to make this better. It's so good things you can do with notifications and the great companion app.  

That it is merely describing the entity I also agree but you also end up on this page when you are clicking the help "?" In automation like me. 
I was just adding a notify in my automation and forgot how to make an critical notification and clicked the  "?" and ended up on the notify page. After that I needed to  digging for the right content because I knew it was somewhere because I used it before. 

A bit confusing is that the companion apps has its own site but that is another story ;)